torch/nn/modules/_wrapped_modules.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
+from torch.nn.modules import Module
+
+r"""Base class for all wrapper modules."""
+class _BaseWrapperModule(Module):
```

Comment:
this class is likely not going to be scriptable - plesae double-check

Reply:
actually I think we probably don't need them to be scriptable since these are not used in graph mode, or are you referring to the case when we quantize in eager mode and then script it?

Reply:
Yes, unless there's a good reason not to, our default should be that everything is scriptable.

In this case, it's a very likely case that I want to quantize using eager mode, tweak my model and script the result